OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -215,6 +228,18 @@ public virtual WVec GetRepulsionForce()
 			return dot >= 0 ? repulsionForce : WVec.Zero;
 		}
 
+		public void DisableRepulsing() {
+			if (Info.Repulsable) {
```

Comment:
The samehere and below: Put the brace of the method on a new line and remove the braces around the single statement.